src/trainer.py

```diff
@@ -961,18 +946,13 @@ def _validate(self, val_pass, tasks, batch_size, periodic_save=True):
 
     def _get_lr(self):
         """ Get learning rate from the optimizer we're using """
-        if self._g_optimizer is not None:
-            lrs = {"global_lr": self._g_optimizer.param_groups[0]["lr"]}
-        else:
-            lrs = {}
-            for task, task_info in self._task_infos.items():
-                lrs["%s_lr" % task] = task_info["optimizer"].param_groups[0]["lr"]
+        lrs = {"global_lr": self._optimizer.param_groups[0]["lr"]}
```

Comment:
Cut this function altogether. It's just making a one-element list for no clear reason now.